pkg/session/tidb.go

```diff
@@ -148,21 +145,7 @@ var (
 // TODO: Remove domap and storeBootstrapped. Use store.SetOption() to do it.
 func ResetStoreForWithTiKVTest(store kv.Storage) {
 	domap.Delete(store)
-	unsetStoreBootstrapped(store.UUID())
-}
-
-func setStoreBootstrapped(storeUUID string) {
-	storeBootstrappedLock.Lock()
-	defer storeBootstrappedLock.Unlock()
-	storeBootstrapped[storeUUID] = true
-}
-
-// unsetStoreBootstrapped delete store uuid from stored bootstrapped map.
-// currently this function only used for test.
-func unsetStoreBootstrapped(storeUUID string) {
-	storeBootstrappedLock.Lock()
-	defer storeBootstrappedLock.Unlock()
-	delete(storeBootstrapped, storeUUID)
+	store.SetOption(StoreBootstrappedKey, nil)
```

Comment:
Could we remove the comment on line145？

Reply:
I left it for next PR of removing domap.